<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addCVanDialog</class>
 <widget class="QDialog" name="addCVanDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add company van</string>
  </property>
  <widget class="QDialogButtonBox" name="addCVanButtonBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>351</width>
     <height>341</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_18">
    <item>
     <widget class="QLabel" name="label_49">
      <property name="text">
       <string>Model:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="model_CV"/>
    </item>
    <item>
     <widget class="QLabel" name="label_50">
      <property name="text">
       <string>License Number:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="licnr_CV"/>
    </item>
    <item>
     <widget class="QLabel" name="label_51">
      <property name="text">
       <string>Company Name:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="comname_CV"/>
    </item>
    <item>
     <widget class="QLabel" name="label_52">
      <property name="text">
       <string>Organization Number:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="orgnr_CV"/>
    </item>
    <item>
     <widget class="QLabel" name="label_54">
      <property name="text">
       <string>Load Capacity:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="loacap_CV">
      <property name="maximum">
       <number>99999</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>addCVanButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>addCVanDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addCVanButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>addCVanDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
